test/blob_test.js

```diff
@@ -1,47 +1,57 @@
-suite.only("Azure Blob", function() {
+suite("Azure Blob", function() {
   var azure   = require('../');
   var assert  = require('assert');
+  var utils   = require('../lib/utils');
 
   // Create azure blob client
   var blob = new azure.Blob({
     accountId: process.env.AZURE_STORAGE_ACCOUNT,
     accessKey: process.env.AZURE_STORAGE_ACCESS_KEY
   });
 
+  var anonymousBlob = new azure.Blob({
+    accountId: process.env.AZURE_STORAGE_ACCOUNT,
+    accountKey: null
+  });
+
+  var containerNamePrefix = 'fast-azure-blob-container';
```

Comment:
It would be great if we could read this from the environment and then maybe do something cool on travis, whereby we encode the changeset of the to-be-tested commit into the container name!  Not important, just a thought :)